src/PKSim.Infrastructure/ProjectConverter/IndividualCalculationMethodsUpdater.cs

```diff
@@ -0,0 +1,75 @@
+﻿using PKSim.Core;
+using PKSim.Core.Model;
+using PKSim.Core.Repositories;
+
+namespace PKSim.Infrastructure.ProjectConverter
+{
+   public interface IIndividualCalculationMethodsUpdater
+   {
+      void AddMissingCalculationMethodsTo(Individual individual);
+      void AddMissingCalculationMethodsTo(Simulation simulation);
+   }
+
+   public class IndividualCalculationMethodsUpdater : IIndividualCalculationMethodsUpdater
+   {
+      private readonly ICalculationMethodRepository _calculationMethodRepository;
+
+      public IndividualCalculationMethodsUpdater(ICalculationMethodRepository calculationMethodRepository)
+      {
+         _calculationMethodRepository = calculationMethodRepository;
+      }
+
+      public void AddMissingCalculationMethodsTo(Individual individual)
+      {
+         if (individual == null)
+            return;
+
+         addMissingCalculationMethods(individual.OriginData, individual.IsHuman);
+      }
+
+      public void AddMissingCalculationMethodsTo(Simulation simulation)
+      {
+         var individual = simulation?.BuildingBlock<Individual>();
+         if (individual == null)
+            return;
+
+         addMissingCalculationMethods(simulation.ModelProperties, individual.IsHuman);
+      }
+
+      private void addMissingCalculationMethods(IWithCalculationMethods withCalculationMethods, bool isHuman)
+      {
+         addRenalAgingCalculationMethodTo(withCalculationMethods, isHuman);
```

Comment:
This should probably be called for individual and simulation so that the simulation has all new calculation methods at all time